README.md

```diff
@@ -21,5 +21,77 @@
 
 Apache APISIX Helm Charts provide the installation of [Apache APISIX](https://github.com/apache/apisix#apache-apisix) components for kubernetes.
 
+Update the dependency chart 
```

Comment:
The following changes should not be added

Reply:
Yes, It has been moved to [Apache APISIX Helm Chart](https://github.com/apache/apisix-helm-chart/tree/master/charts/apisix)
@TkClark Please have a look.